db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+message_1 = Message.create(message: 'Salam there')
+message_2 = Message.create(message: 'Hikya vun')
+message_3 = Message.create(message: 'Hi there, how are you?')
+message_4 = Message.create(message: 'I am a developer')
+message_5 = Message.create(message: 'My name is Elmar')
```

Comment:
- [ ] Currently, I'm getting an error while running the seed file. "**unknown attribute 'message' for Message.**". I've seen that you are using `greeting` column in messages table so the statement should be like this.

```
message_1 = Message.create(greeting: 'Salam there')
```

Reply:
Fixed

Reply:
- [ ] As mentioned by the previous reviewer, kindly use his method it is working well.

Instead of `message_1 = Message.create(message: 'Salam there')` 

![image](https://user-images.githubusercontent.com/55635977/155835842-228b701f-2bb4-4928-8630-06fc902bc6b8.png)


use

`message_1 = Message.create(greeting: 'Salam there')`

it will work.